ocictl/cmd/build.go

```diff
@@ -109,7 +115,7 @@ func (b *buildCmd) run(args []string) error {
 	case v1alpha1.BuildahFramework:
 		{
 			b := buildah.Buildah{
-				Logger:        common.GetLogger(b.debug),
+				Logger:        logger,
```

Comment:
should these be reordered for better readability 

```
StorageDriver: b.storageDriver,
Logger:        logger,
```

Reply:
Can be, I don't have a strong preference. Currently it's alphabetical, but if we feel it makes more sense the other way then i can make this change.